docs/man/git-lfs-migrate.1.ronn

```diff
@@ -84,13 +84,39 @@ options and these additional ones:
     Write to 'path' a file with the mapping of each rewritten commits. The file
     format is CSV with this pattern: `OLD-SHA`,`NEW-SHA`
 
-If `--include` or `--exclude` (`-I`, `-X`, respectively) are given, the
+* `--no-rewrite`
+    Migrate large objects to Git LFS in a new commit without rewriting git history.
+    Please note that when this option is used, the `migrate import` command will expect
+    a different argument list, specialized options will become available, and the core 
+    `migrate` options will be ignored. See [IMPORT (NO REWRITE)].
+
+If `--no-rewrite` is not provided and `--include` or `--exclude` (`-I`, `-X`, respectively) are given, the
```

Comment:
Nit: it looks like a few lines in this file are longer than 80 columns. Can you make sure that the wrapping is set properly for `.ronn` files?